src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -333,6 +339,30 @@ debug_configuration::debug_configuration()
         }
     }
 
+    if (dump_runtime_memory_pool_iters_str.size() > 0) {
+        // eliminate '"' from string to avoid parasing error
+        dump_runtime_memory_pool_iters_str.erase(
+            std::remove_if (dump_runtime_memory_pool_iters_str.begin (), dump_runtime_memory_pool_iters_str.end (), [](char c)
+                 {
+                    return c == '\"';
+                 }),
+                 dump_runtime_memory_pool_iters_str.end ());
+        if (dump_runtime_memory_pool_iters_str.size() > 0) {
+            dump_runtime_memory_pool_iters_str = " " + dump_runtime_memory_pool_iters_str + " ";
+            std::istringstream ss(dump_runtime_memory_pool_iters_str);
+            std::string token;
+            while (ss >> token) {
+                try {
+                    dump_runtime_memory_pool_iters.insert(static_cast<int64_t>(std::stol(token)));
+                } catch(const std::exception& ex) {
+                    dump_runtime_memory_pool_iters.clear();
+                    GPU_DEBUG_COUT << "OV_GPU_DumpRuntimeMemoryPoolIters was ignored. It cannot be parsed to integer array." << std::endl;
+                    break;
+                }
+            }
```

Comment:
could you refactor this code a bit to have something like `std::vector<int> parse_int_array(const char *string)`?

Reply:
Done